src/isIsogram.test.js

```diff
@@ -8,12 +8,18 @@ describe(`Function 'isIsogram':`, () => {
   });
 
   it(`should return a boolean`, () => {
-
+    expect(typeof isIsogram('word')).toBe('boolean');
   });
 
-  it(`should be case sensitive`, () => {
+  it(`should return 'true' if the string is empty`, () => {
+    expect(isIsogram('')).toBeTruthy();
+  });
 
+  it(`should return 'true' if 'Omit' is used`, () => {
+    expect(isIsogram('Omit')).toBe(true);
   });
 
-  // write more tests here
+  it(`should return 'false' if 'deterrent' is used`, () => {
+    expect(isIsogram('deterrent')).toBe(false);
+  });
 });
```

Comment:
You shouldn't have deleted the case sensitivity test! Try to think how to modify it so that it checks case sensitivity